zebra/if_netlink.c

```diff
@@ -1371,14 +1371,14 @@ int netlink_link_change(struct nlmsghdr *h, ns_id_t ns_id, int startup)
 
 			if_handle_vrf_change(ifp, vrf_id);
 		} else {
-			bool was_bridge_slave, was_bond_slave;
+			bool was_bridge_secondary, was_bond_secondary;
```

Comment:
was_bridge_member.  I believe was_bond_secondary captures it's tech meaning pretty good

Reply:
Actually, member is used to refer to both in the header file. I will change all these to "member"